examples/eks/manager.yaml

```diff
@@ -0,0 +1,275 @@
+apiVersion: v1
```

Comment:
master is missing this update after manager merge. Please extract it to separate commit

Reply:
removed